index.js

```diff
@@ -231,217 +231,197 @@ smartapp
         console.log(`Button${DeviceID} Sensor: ${signal} @${LocationID}`);
     })
 
-// Closes any open session and resets state for the given location
-async function resetSession(locationid){
-  try{
-    if(await db.closeSession(locationid)){
-      let session = await db.getMostRecentSession(locationid);
-      console.log(session);
-      await db.updateSessionResetDetails(session.sessionid, "Manual reset", "Reset");
-      await redis.addStateMachineData("Reset", locationid);
-    }
-    else{
-      console.log("There is no open session to reset!")
-    }
-  }
-  catch {
-    console.log("Could not reset open session");
-  }
-}
-
```

Comment:
I think we used reset session when a button on the front end was clicked. I wonder if the link is broken :?

Reply:
Actually, we removed this functionality from the front end interface. So removing this function makes sense. When we do session resets through code, autoResetSession is used instead. The naming of that function doesn't make sense since though it is called automatically by the code, functionality is still session reset (We did have separate functions to have different chatbot/session notes)

Reply:
Thanks for explaining! This must have been why I had the vague feeling that reset wasn't being used. Makes sense.